src/main/java/com/streamingplatform/moviestreamingplatform/model/dto/RoleDto.java

```diff
@@ -0,0 +1,12 @@
+package com.streamingplatform.moviestreamingplatform.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Data;
+
+@Data
+public class RoleDto {
+
+    @JsonProperty("name")
```

Comment:
@Ruleabogdan this is redundant. Please remove all jsonProperty annotations that are identical to the java name